ide/parsing.indexing/src/org/netbeans/modules/parsing/impl/indexing/PathRegistry.java

```diff
@@ -68,7 +68,7 @@
 public final class PathRegistry implements Runnable {
 
     private static final boolean FIRE_UNKNOWN_ALWAYS = false;
-    /*test*/ static final int FIRER_EVT_COLLAPSE_WINDOW = 500;
+    /*test*/ static final int FIRER_EVT_COLLAPSE_WINDOW = Integer.getInteger(PathRegistry.class.getName()+".FIRER_EVT_COLLAPSE_WINDOW", 500);
```

Comment:
Just an idea: it might be good to add a comment here, that this is a system property to help tests, so that noone ever can claim this is some supported system property we have to retain even if the unittests run fast without twiddling with ti.

Reply:
tbh I was wondering if reducing this value in production (e.g to 200) would have any effect. Who knows what else is waiting for the delay to complete. Maybe certain editor features would refresh faster. 500ms is fairly long in computer science.

Reply:
Btw I wanted to make it private but it is actually used by a static inner class (and not in tests). So the /*test*/ comment was basically false advertisement until now :)

I guess I could make it private but add a non-public getter and add a proper comment.